packages/yoroi-extension/app/api/ada/lib/storage/bridge/utils.js

```diff
@@ -331,14 +331,11 @@ export function isJormungandrAddress(
   kind: CoreAddressT
 ): boolean {
   // note: excluding legacy byron addresses
-  const types = [
-    CoreAddressTypes.JORMUNGANDR_SINGLE,
-    CoreAddressTypes.JORMUNGANDR_GROUP,
-    CoreAddressTypes.JORMUNGANDR_ACCOUNT,
-    CoreAddressTypes.JORMUNGANDR_MULTISIG,
-  ];
-
-  return kind in types;
+  if (kind === CoreAddressTypes.JORMUNGANDR_SINGLE) return true;
+  if (kind === CoreAddressTypes.JORMUNGANDR_GROUP) return true;
+  if (kind === CoreAddressTypes.JORMUNGANDR_ACCOUNT) return true;
+  if (kind === CoreAddressTypes.JORMUNGANDR_MULTISIG) return true;
+  return false;
 }
 
 export function isErgoAddress(
```

Comment:
I think this solution is more cleaner 
```suggestion
const types = [
    CoreAddressTypes.JORMUNGANDR_SINGLE,
    CoreAddressTypes.JORMUNGANDR_GROUP,
    CoreAddressTypes.JORMUNGANDR_ACCOUNT,
    CoreAddressTypes.JORMUNGANDR_MULTISIG,
  ];
  return  types.some(addrType => addrType === kind);
}
```

Reply:
@AhmedIbrahim336 or you can use `types.includes(kinds)`